ui/lib/utils/telemetry.ts

```diff
@@ -8,15 +8,36 @@ export async function init() {
   // check option
   const res = await client.getInstance().getInfo()
   if (res?.data?.disable_telemetry === false) {
-    mixpanel.opt_in_tracking()
     // https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelset_config
     mixpanel.set_config({
       batch_requests: true,
       persistence: 'localStorage',
     })
+    mixpanel.register({
+      // $initial_referrer and $initial_referring_domain overrides don't work in register()
```

Comment:
initial_referrer is registered using register_once, so that it cannot be overridden. You can use property blacklist: https://help.mixpanel.com/hc/en-us/articles/115004601523-Block-Default-Mixpanel-Properties